documentation/Fukushima.tex

```diff
@@ -0,0 +1,131 @@
+
+\subsection*{Integrals of the second kind for $m$ close to $1$}
+Fukushima defines $B\of{m}$ as:
+\[
+B\of{m} = \frac{E\of{m} - m_c K\of{m}}{m}
+\]
+This expression can be rewritten using the definitions above:
+\begin{align*}
+B\of{m} &= \frac{1}{m}\pa{E_X\of{m_c} X\of{m_c} + E_0\of{m_c} - m_c K_X\of{m_c} X\of{m_c}}\\
+&= \frac{1}{m}\pa{X\of{m_c}\pa{E_X\of{m_c} - m_c K_X\of{m_c}} + \frac{1}{2 K_X\of{m_c}}}
+\end{align*}
+Similarly the definition of $D\of{m}$:
+\[
+D\of{m} = \frac{K\of{m} - E\of{m}}{m}
+\]
+can be rewritten as:
+\begin{align*}
+D\of{m} &= \frac{1}{m}\pa{K_X\of{m_c} X\of{m_c} - E_X\of{m_c} X\of{m_c} - E_0\of{m_c}}\\
+&= \frac{1}{m}\pa{X\of{m_c}\pa{K_X\of{m_c} - E_X\of{m_c}} - \frac{1}{2 K_X\of{m_c}}}
+\end{align*}
+These formulæ provide a means to compute $B\of{m}$ and $D\of{m}$ for $m$ close to $1$.  First, a polynomial approximation of $q\of{m_c}$
+is computed, whose leading term is of order $m_c / 16$.  Then the $\log$ of that approximation is evaluated, yielding $X\of{m_c}$ (this
+is the part that carries the logarithmic singularity).  Finally, $E_X\of{m_c}$ and $K_X\of{m_c}$ are computed using
+Taylor or Maclaurin approximations.
+
+It is easy to see that $B\of{m} = X\of{m_c} K_X\of{m_c} - D\of{m}$, which provides a simpler formula for computing $B\of{m}$ once 
+$D\of{m}$ is known.
+
+\subsection*{Integrals of the second kind when $m$ tends towards $1$}
+We are now interested in computing the leading term of $B\of{m}$ and $D\of{m}$ when $\conv m 1$.  First, we have $B\of{1} = 1$ 
+([Fuku11a], equation 1.9).  However, $\conv {D\of{m}} +\infty$ when $\conv m 1$.  To deal with this singularity we recall that:
+\[
+B\of{m} = X\of{m_c} K_X\of{m_c} - D\of{m}
+\]
+thus:
+\[
+D\of{m} = X\of{m_c} K_X\of{m_c} - B\of{m}
+\]
+Remember that $X\of{m_c} = -\log q\of{m_c}$ and that $q\of{m_c} = m_c / 16 + \BigO\of{m_c^2}$.  Therefore:
+\[
+X\of{m_c} = \log 16 - \log\of{m_c} + \BigO\of{m_c^2}
+\]
+Furthermore $K\of{0} = \gp / 2$, so $K_X\of{0} = 1/2$.  Putting all these relations together we obtain the following equation:
+\[
+D\of{m} = 2 \log 2 - 1 - \frac{\log m_c}{2} + \BigO\of{m_c^2}
+\]
+
+\subsection*{Integrals of the second kind for $m$ close to $0$}
+The expressions defined in the first section can be rewritten by changing the variable to be $m = 1 - m_c$.  In particular:
+\begin{align*}
+E_X\of{m} &= \pa{1 - \frac{E\of{m}}{K\of{m}}K_X\of{m}}\\
+&= \frac{1}{\gp}\pa{K\of{m} - E\of{m}}
+\end{align*}
+Thus:
+\[
+D\of{m} = \frac{\gp}{m}E_X\of{m}
+\]
+Similarly, define $B_X^*\of{m}$ as follows:
+\[
+B_X^*\of{m} = E_X\of{m} - m K_X\of{m}
+\]
+We have:
+\begin{align*}
+B_X^*\of{m} &= \frac{1}{\gp}\pa{K\of{m} - E\of{m} - m K\of{m}}\\
+&= \frac{1}{\gp}\pa{m_c K\of{m} - E\of{m}}\\
+&= -\frac{m B\of{m}}{\gp}
+\end{align*}
+Therefore:
+\[
+B\of{m} = -\frac{\gp}{m} B_X^*\of{m}
+\]
+These formulæ make it possible, by computing a Maclaurin approximation of $B_X^*\of{m}$ and $E_X\of{m}$, to evaluate $B\of{m}$
+and $D\of{m}$ for $m$ close to $0$.
```

Comment:
Consider adding a one-sentence conclusion comparing this method with the one described by Fukushima, so that we understand why this is done that way.

Reply:
Added a "Conclusion" section.

Reply:
As discussed, clarify the conclusion to actually explain the differences between the two methods.